src/Tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, ReactNode } from 'react'
+import { Text } from '../Text'
+import { Container, Tip } from './Tooltip.styles'
+
+export type TooltipPosition = 'top' | 'bottom' | 'left' | 'right'
+
+export interface TooltipProps {
+  children: ReactNode
+  title: string
+  content: string | ReactNode
+  position: TooltipPosition
+  underline?: boolean
+}
+
+export const Tooltip: FC<TooltipProps> = ({
+  children,
+  title,
+  content,
+  position,
```

Comment:
should we default this to top since it's probably the most commonly used?

Also based on Figma layout proposal I'm wondering how we are going to handle these? I know that Alistair is off so might be better to wait for him or to ask in the design-system channel about clarification?

- The positioning of the tooltip relative to the page is determined by the amount of space it has available 
- Tooltip is agnostic to the layout of the page and it’s surrounding elements, but it’s best to accommodate it in the whitespace

Reply:
About to drop a message in design-system, will update when I get a response :)